### PR TITLE
DM-13943: Deprecate VisitInfo.getExposureId()

### PR DESCRIPTION
**Pull request by kfindeisen**
This PR removes uses of `VisitInfo.getExposureId()`, which was deprecated in lsst/afw#614.

### CODE REVIEW

File: python/lsst/cp/pipe/ptc/measurePtcGen2Task.py [hunk 1]
```diff
@@ -124,7 +124,7 @@ def runDataRef(self, dataRefList):
                 self.log.warn("postISR exposure could not be retrieved. Ignoring flat.")
                 continue
             expList.append(tempFlat)
-        expIds = [exp.getInfo().getVisitInfo().getExposureId() for exp in expList]
+        expIds = [exp.info.getVisitInfo().id for exp in expList]
```
Comment (spenczar) [inline python/lsst/cp/pipe/ptc/measurePtcGen2Task.py:127]: Should this be `exp.info.id for exp in expList`?

File: python/lsst/cp/pipe/utils.py [hunk 2]
```diff
@@ -688,7 +688,7 @@ def arrangeFlatsByExpId(exposureList, exposureIdList):
     """
     flatsAtExpId = {}
     # sortedExposures = sorted(exposureList, key=lambda exp:
-    # exp.getInfo().getVisitInfo().getExposureId())
+    # exp.info.getVisitInfo().id)
```
Comment (spenczar) [inline python/lsst/cp/pipe/utils.py:691]: What's going on here?

**Review by spenczar — CHANGES_REQUESTED** [review]

File: python/lsst/cp/pipe/ptc/measurePtcGen2Task.py (hunk 1 above)
Comment (kfindeisen) [reply python/lsst/cp/pipe/ptc/measurePtcGen2Task.py:127]: Oops, I could swear I included an explanation in the commit message (I'll go back and add one).

@czwa told me that these two identifiers were supposed to be the visit ID and not the per-detector ID, so basically the use of `getExposureId()` here was a bug.

File: python/lsst/cp/pipe/utils.py (hunk 2 above)
Comment (czwa) [reply python/lsst/cp/pipe/utils.py:691]: The comment was written too long, and was split when comment line length linting was enabled (part of DM-30245).  It should probably be removed, as it's no longer useful.

**Review by spenczar — APPROVED** [review]

**Pull request merged**